addons/account/models/account_move.py

```diff
@@ -3202,7 +3202,11 @@ def _get_move_display_name(self, show_ref=False):
 
     def _get_reconciled_amls(self):
         """Helper used to retrieve the reconciled move lines on this journal entry"""
-        reconciled_lines = self.line_ids.filtered(lambda line: line.account_id.account_type in ('asset_receivable', 'liability_payable'))
+        reconciled_lines_cache = self.env.context.get('reconciled_lines_cache', {}.fromkeys(self.ids))
```

Comment:
I don't get what is calling this method. Your commit message is explaining what you do but not why you are doing this in this particular method

Reply:
@smetl the reason is in the flamegraph